packages/aws-cdk-lib/aws-batch/lib/job-queue.ts

```diff
@@ -135,6 +143,79 @@ export interface OrderedComputeEnvironment {
   readonly order: number;
 }
 
+/**
+ * Specifies an action that AWS Batch will take after the job has remained at
+ * the head of the queue in the specified state for longer than the specified time.
+ */
+
+export interface JobStateTimeLimitAction {
+  /**
+ * The action to take when a job is at the head of the job queue in the specified state
+ * for the specified period of time.
+ */
+  readonly action: JobStateTimeLimitActionsAction;
```

Comment:
It may be better to make `action` optional and set the default value to `JobStateTimeLimitActionsAction.CANCEL`, since that is the only value we can configure at the moment.

Reply:
Thank you for your comment.
I think your suggestion is better.

I have changed action to be optional and set its default value to JobStateTimeLimitActionsAction.CANCEL.